lib/utils/isFirstNodeOfRoot.js

```diff
@@ -1,6 +1,11 @@
+// @ts-nocheck
```

Comment:
I think we need some explanation here as well

Reply:
https://github.com/stylelint/stylelint/pull/4387#discussion_r339769695 😇